Sources/MongoSwift/MongoCollection+Indexes.swift

```diff
@@ -284,7 +284,8 @@ extension MongoCollection {
         let opts = try BSONEncoder().encode(options)
         let reply = Document()
         var error = bson_error_t()
-        if !mongoc_collection_write_command_with_opts(self._collection, command.data, opts?.data, reply.data, &error) {
+        guard mongoc_collection_write_command_with_opts(self._collection, command.data, opts?.data, reply.data, &error)
+        else {
```

Comment:
Was `else` put on a new line by the linter? If it's being wrapped I might have expected some indentation here, although I honestly haven't encountered this case before so I'm not sure how linting handles it.

Curious if the syntax from `MongoCollection+Read.swift` may be the way to go. For example:

```
guard mongoc_collection_read_command_with_opts(
    self._collection, command.data, rp, opts?.data, reply.data, &error) else {
```

Reply:
this is something else I want a linter rule for... I am rather inconsistent and sometimes break after the open parens and sometimes break mid argument list. but I think the former is probably better.

Reply:
Yeah, this was something that the linter did not care about. My guess is that this is because it is seeing it the same way it sees `if` and `else`:
```
if { ... }
else { ... }
```
Where `if` and `else` is aligned. But I like the syntax you've shown from `MongoCollection+Read.swift` better.